test/cctest/node_test_fixture.h

```diff
@@ -41,8 +41,8 @@ struct Argv {
     snprintf(argv_[0], prog_len, "%s", prog);
     snprintf(argv_[0] + prog_len, arg1_len, "%s", arg1);
     snprintf(argv_[0] + prog_len + arg1_len, arg2_len, "%s", arg2);
-    argv_[1] = argv_[0] + prog_len + 1;
-    argv_[2] = argv_[0] + prog_len + arg1_len + 1;
+    argv_[1] = argv_[0] + prog_len;
```

Comment:
this appears completely unrelated, is it just a fix for something you noticed was broken? if so, it should go in its own PR, so it can be backported and semvered independently